python_benchmarks/benchmark_layernorm.py

```diff
@@ -0,0 +1,201 @@
+import pytest
```

Comment:
I think the convention is to name the benchmark files `test_*.py` since they are valid pytest tests and can be mixed with pure tests. This way, pytest will be able to find them automatically if you run `pytest python_benchmarks/`.